neurom/io/hdf5.py

```diff
@@ -81,7 +81,7 @@ class H5(object):
     def read_v1(filename):
         '''Read an HDF5 v1 file and return a tuple of data, offset, format.'''
         points, groups = _unpack_v1(h5py.File(filename, mode='r'))
-        data = H5.unpack_data(points, groups)
+        data = H5.unpack_data(*H5.remove_duplicate_points(points, groups))
```

Comment:
YES, this is already optional in the upcoming commit... Just fixing tests before sending everything up.